docs/docker/run.md

```diff
@@ -1,41 +1,39 @@
-## Run you own __Cardano Node__ 
-
-## OS Requirements
+### OS Requirements
 
 - "docker-ce" installed.
 
-## Private mode
+### Private mode
 
-### use cases
+#### Use Cases
 
 - Pool Management
 - Wallet Management
 - Node testing
 
 ```bash
-docker run -dit 
--e NETWORK=mainnet 
---name <YourCName>
--p <your_custom_path>:/opt/cardano/cnode/priv
--p <your_custom_db_path>:/opt/cardano/cnode/db
+docker run -dit \
+-e NETWORK=mainnet \
+--name <YourCName> \
+-p <your_custom_path>:/opt/cardano/cnode/priv \
```

Comment:
@rdlrt @redoracle the -p flag is for exposing ports. The -v flag is for binding a docker volume or external path. I believe line 17 & 18 should instead be:
```
-v <your_custom_path>:/opt/cardano/cnode/priv \
-v <your_custom_db_path>:/opt/cardano/cnode/db \
```

Reply:
Resolved on base PR